gettsim/parameters/kinderzuschl.yaml

```diff
@@ -47,7 +47,7 @@ kinderzuschlag_min_eink_alleinerz:
   2008-10-01:
     scalar: 600
     reference: Art. 1 G. v. 24.09.2008 BGBl. I S. 1854.
-kinderzuschlag_transferentzug_eltern:
+kinderzuschl_transferentzug_eltern:
```

Comment:
There are other deal breakers, too. And I think `kinderzuschl` isn't too bad.

Reply:
Well, there are always tradeoffs.

Are the other deal breakers the same two as  in this list?

https://github.com/iza-institute-of-labor-economics/gettsim/pull/314#issuecomment-982452673

In that case, my vote would be for making those private.

Reply:
Yes, `kinderzuschlag_vermögens_check_hh` and `wohngeld_kinderzuschlag_vorrang_hh`.

In line with GEP-01 we shouldn't create purely internal columns if there is an appropriate other solution. And I see almost no disadvantage of `kinderzuschl`. I find it is very obvious what is meant and it fits nicely to the scheme of  `arbeitsl_geld` or `ges_rentenvers`.

So my vote would be for the abbreviation. Then we only needed to make `zu_verst_eink_kein_kinderfreib_tu` private.

Reply:
As I said, there are trade-offs. For example, when talking about the code, having pronounceable names is helpful. I do not see much of a drawback in making those two variables private, who would ever want to use them in Stata code?

But I guess the votes are 1:1, so keep some coming, please! E.g. @MaxBlesch @mjbloemer @Eric-Sommer 

Btw, analogously to `günstiger_kinderfreib_tu`, I think that `vorrang_wohngeld_kinderzuschlag_hh` would be more intuitive.

Reply:
But in that case you preferred `kinderfreib_günstiger_tu`, right?

>Almost -- I would prefer kinderfreib_günstiger_tu

Which order do you prefer here? For me, both are fine, with small preferences for the current version.

Reply:
:rofl: 

Sorry bout the noise.

Reply:
Just to make it easier on the decisive voters, let me repeat the issue after messing up the thread just above.

Problem: We cannot keep the name `kinderzuschlag` and all variables derived from it due to length issues. 2 potential solutions:

- @ChristianZimpelmann suggests renaming `kinderzuschlag` to `kinderzuschl` everywhere.
 
  *very obvious what is meant and it fits nicely to the scheme of  `arbeitsl_geld` or `ges_rentenvers`.*
- @hmgaudecker suggests making `kinderzuschlag_vermögens_check_hh` and `wohngeld_kinderzuschlag_vorrang_hh` private. 

  *when talking about the code, having pronounceable names is helpful. I do not see much of a drawback in making those two variables private, who would ever want to use them in Stata code?*

Please weigh in, people! Thinking of @Eric-Sommer @mjbloemer @janosg @MaxBlesch, but anyone else feel free to say something, too! Please do so somewhat quickly - this PR has grown so large that we should resolve it soon, very inefficient to go back to it multiple times.

Reply:
I think I prefer @ChristianZimpelmann's solution of the two above, because I have a preference for as many public columns as possible. 

However, I think abbreviations in the middle of a name with "_" as a replacement are the best way to save characters and keep things pronounceable. If it is about saving two characters in both names, I would prefer changing to "kind_zuschlag" and then "kind_zuschlag_vermögen_check_hh" and "wohngeld_kind_zuschlag_prio_hh". Maybe change "vorrang" everywhere to "prio". Much shorter and for me expresses the same!

Reply:
2:1 for `kinderzuschl`. Very good, let's leave it at that!!!